play-ebean/src/main/java/play/db/ebean/DefaultEbeanConfig.java

```diff
@@ -232,8 +232,8 @@ private static Reflections getReflections(Environment env, String packageName) {
         private static ConfigurationBuilder getReflectionsConfiguration(String packageName, ClassLoader classLoader) {
             return new ConfigurationBuilder()
                     .addUrls(ClasspathHelper.forPackage(packageName, classLoader))
-                    .filterInputsBy(new FilterBuilder().includePackage((packageName + ".").replace(".","\\.") + ".*"))
-                    .setScanners(new TypeElementsScanner(), new TypeAnnotationsScanner(), new SubTypesScanner());
+                    .filterInputsBy(new FilterBuilder().includePackage(packageName))
+                    .setScanners(new TypeElementsScanner(), Scanners.TypesAnnotated, Scanners.SubTypes);
```

Comment:
`TypeAnnotationsScanner` and `SubTypesScanner` are deprecated and use `TypesAnnotated` and `SubTypes` in constructors respectively